server/server.go

```diff
@@ -353,15 +353,22 @@ func New(
 	}
 
 	if s.seccompEnabled {
-		seccompProfile, fileErr := ioutil.ReadFile(config.SeccompProfile)
-		if fileErr != nil {
-			return nil, fmt.Errorf("opening seccomp profile (%s) failed: %v", config.SeccompProfile, fileErr)
-		}
-		var seccompConfig seccomp.Seccomp
-		if jsonErr := json.Unmarshal(seccompProfile, &seccompConfig); jsonErr != nil {
-			return nil, fmt.Errorf("decoding seccomp profile failed: %v", jsonErr)
+		if config.SeccompProfile != "" {
+			seccompProfile, fileErr := ioutil.ReadFile(config.SeccompProfile)
+			if fileErr != nil {
+				return nil, fmt.Errorf("opening seccomp profile (%s) failed: %v",
+					config.SeccompProfile, fileErr)
+			}
+			logrus.Infof("Using seccomp profile %q", config.SeccompProfile)
+			var seccompConfig seccomp.Seccomp
+			if jsonErr := json.Unmarshal(seccompProfile, &seccompConfig); jsonErr != nil {
+				return nil, fmt.Errorf("decoding seccomp profile failed: %v", jsonErr)
+			}
+			s.seccompProfile = &seccompConfig
+		} else {
+			logrus.Infof("No seccomp profile specified, using the internal default.")
```

Comment:
Nit of a nit: remove the trailing dot for consistency

Reply:
Affirmative.